docs/onboarding.md

```diff
@@ -0,0 +1,57 @@
+---
+title: How to get started with WorkflowHub
+---
+
+WorkflowHub will help make your workflows findable and citable, and this guide will help you get started with that process.
+
+## 1. [Register yourself on WorkflowHub](https://workflowhub.eu/signup)
+
+{% include callout.html type="note" content="you can also log in using your GitHub credentials." %}
```

Comment:
In addition to GitHub, we should also mention [LS Login](https://lifescience-ri.eu/ls-login/).

I think this should work:
```
{% include callout.html type="note" markdown="span"
content="You can also log in using your [LS Login](https://lifescience-ri.eu/ls-login/) or [GitHub](https://github.com/) credentials." %}

```